main.tf

```diff
@@ -81,6 +84,20 @@ data "aws_iam_policy_document" "default" {
       identifiers = ["*"]
     }
   }
+
+  # Support deployment ARNs
+  statement {
+    actions = ["s3:*"]
```

Comment:
It will allow also bucket-related operations (CREATE, DELETE, etc.).
Should we stick only to [Amazon S3 Permissions for Object Operations](http://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html)?

Reply:
I have the same question.
We need those broad permissions only if the external systems will need to be able to also create and delete buckets and objects in them.
Maybe it's what is needed for systems like `CircleCI`?

Reply:
Agreed, how restrictive can we go? http://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html#example-bucket-policies-use-case-1

Reply:
I also found this:


> Important
> When testing permissions using the Amazon S3 console, you will need to grant additional permissions that the console requires—s3:ListAllMyBuckets, s3:GetBucketLocation, and s3:ListBucket permissions. For an example walkthrough that grants permissions to users and tests them using the console, see [An Example Walkthrough: Using user policies to control access to your bucket](http://docs.aws.amazon.com/AmazonS3/latest/dev/walkthrough1.html)